### PR TITLE
CI Update

### PR DESCRIPTION
**Pull request by v-alje**
Build.Reason:Manual by Alma Jenks
 Build.Url:https://apidrop.visualstudio.com/Content%20CI/_build/results?buildId=249591&view=results
 source_repo.branch:release-msbuild
 source_repo.url:https://apidrop.visualstudio.com/_git/binaries

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

**Review by jkoritzinsky — APPROVED** [review]
System.Runtime.InteropServices LGTM

File: xml/Microsoft.Build.Tasks/CreateProperty.xml [hunk 1]
```diff
@@ -161,8 +161,8 @@
                         Output TaskParameter="Value" PropertyName="MyTargetsToBuild"
                     /CreateProperty
             
-            We need to respect the semicolon that they put in the value, and need to treat
-            this exactly as if they had done:
+            We need to respect the semicolon that he put in the value, and need to treat
```
Comment (gewarren) [inline xml/Microsoft.Build.Tasks/CreateProperty.xml:164]: I would revert this file.

**Review by gewarren — APPROVED** [review]

File: xml/Microsoft.Build.Tasks/CreateProperty.xml (hunk 1 above)
Comment (v-alje) [reply xml/Microsoft.Build.Tasks/CreateProperty.xml:164]: @ghogen It looks like two of the MSBuild changes revert a bulk pronoun change from a few days ago. Should these be updated in the MSBuild source code?

**Review by maryamariyan — APPROVED** [review]
Microsoft.Extensions.* LGTM

File: xml/Microsoft.Build.Tasks/CreateProperty.xml (hunk 1 above)
Comment (ghogen) [reply xml/Microsoft.Build.Tasks/CreateProperty.xml:164]: Yes, but for now let's make the change here to pass this PR.
```suggestion
            We need to respect the semicolon that they put in the value, and need to treat
```

File: xml/Microsoft.Build.Tasks/CreateProperty.xml [hunk 2]
```diff
@@ -161,8 +161,8 @@
...
+            this exactly as if he had done:
```
Comment (ghogen) [inline xml/Microsoft.Build.Tasks/CreateProperty.xml:165]:
```suggestion
            this exactly as if they had done:
```

Comment (ghogen) [reply xml/Microsoft.Build.Tasks/CreateProperty.xml:165]: .NET docs did a pass for inclusive pronouns, so adding that change to the original source comments as well
@rainersigwald

**Pull request merged**